.github/workflows/release.yml

```diff
@@ -121,8 +121,50 @@ jobs:
         name: dist
         path: dist/
 
+  build-linux-aarch64:
```

Comment:
Perhaps move the x86 Linux build into this step as well?

Reply:
@elprans, I have done the changes as per your suggestion. Please have a look. Thanks in advance.